examples/2d/plot_sklearn.py

```diff
@@ -0,0 +1,43 @@
+"""
```

Comment:
Needs to be formatted for the gallery like the other examples.

Reply:
:+1:

Reply:
It is, isnt it? e.g. https://github.com/kymatio/kymatio/blob/master/examples/2d/long_mnist_classify.py

Reply:
Don't you need the long line of `#`s to mark off the text portions (instead of having them appear as just comments in the code)? I don't quite remember.

Reply:
aah ok I see. I thought were referring to the top part of the file.

Reply:
I'd like to see the result (on the website) but looks good